InfobipClient/InfobipClientLib/InfobipClientLib.nuspec

```diff
@@ -2,15 +2,15 @@
 <package >
   <metadata>
     <id>Infobip.Api.Client</id>
-    <version>1.1.0</version>
+    <version>1.2.0</version>
     <title>Infobip API C# Client Library</title>
     <authors>Infobip</authors>
     <owners>Infobip</owners>
     <licenseUrl>https://github.com/infobip/infobip-api-csharp-client</licenseUrl>
     <projectUrl>https://github.com/infobip/infobip-api-csharp-client</projectUrl>
     <requireLicenseAcceptance>false</requireLicenseAcceptance>
     <description>Infobip API C# Client Library</description>
-    <releaseNotes>Added OMNI API support</releaseNotes>
+    <releaseNotes>Added India DLT parameters support</releaseNotes>
     <copyright>Copyright 2017</copyright>
```

Comment:
Can you please update the copyright year as well?
(I know it's irrelevant for this change, but it sticks out and should make this release a bit nicer.)

Reply:
Done!